js/src/util/index.js

```diff
@@ -153,6 +153,14 @@ const isVisible = element => {
   return false
 }
 
+const isDisabled = element => {
+  if (!element) {
+    return true
```

Comment:
I'd return false here.

Reply:
I am not sure. In code if an element not exists, you don't want to count it as valid. right?

Reply:
But the check is for `isDisabled`. If the element doesn't exist then it can't be disabled?

Reply:
Usually we check if a button/anchor/input (or another div) is disabled. Almost in any case it will not be null. The  null check is for the 'just in case'  situation

Reply:
I prefer being stricter :) That being said, the logic seems off to me for the aforementioned reason.

Reply:
IMO, if the DOM element does not exist it should not be considered as enabled. The absence of the DOM element is also like the disability :thinking: